components/list/list-item.js

```diff
@@ -182,6 +198,17 @@ class ListItem extends RtlMixin(LitElement) {
 	render() {
 		const contentId = getUniqueId();
 		const checkBoxId = getUniqueId();
+		const link = html`
+			<a class="d2l-list-item-link"
+				?download="${this.download}"
+				href="${ifDefined(this.href)}"
+				hreflang="${ifDefined(this.hreflang)}"
+				rel="${ifDefined(this.rel)}"
+				target="${ifDefined(this.target)}"
+				type="${ifDefined(this.type)}">
```

Comment:
Again, seems like way overkill to expose every link property up to consumers. And we could get collisions -- `type` on `<a>` is different from `type` on `input`. I say we only expose `href` and be done with it.